NEWS.md

```diff
@@ -1,67 +1,94 @@
 # scoringutils 1.2.2.9000
 
-This major update and addresses a variety of comments made by reviewers from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)).
+This update represents a major rewrite of the package and introduces breaking changes. If you want to keep using the older version, you can download it using `remotes::install_github("epiforecasts/scoringutils@v1.2")`. 
 
-The update introduces breaking changes. If you want to keep using the older version, you can download it using `remotes::install_github("epiforecasts/scoringutils@v1.2")`. 
+This update was done to address comments made by reviewers from the Journal of Statistical Software (see preprint of the manuscript [here](https://arxiv.org/abs/2205.07090)). The most comprehensive documentation for the new package after the rewrite is the [revised version](https://drive.google.com/file/d/1URaMsXmHJ1twpLpMl1sl2HW4lPuUycoj/view?usp=drive_link)
```

Comment:
I think this is quite underwhelming as a justification. I would focus on the major changes and the reasons behind them (i.e to make it more modular). 

I also thought we were giving examples of common use cases eetc in this news post? Or are you thinking about that as another issue?

Reply:
i.e. below just has loads of specific details and there is little sense of the overall plan/purpose

Reply:
yeah definitely, good point